user_sessions/models.py

```diff
@@ -1,29 +1,18 @@
 from django.conf import settings
+from django.contrib.sessions.base_session import (
+    AbstractBaseSession, BaseSessionManager,
+)
 from django.db import models
-from django.utils.translation import gettext_lazy as _
 
 from .backends.db import SessionStore
 
 
-class SessionManager(models.Manager):
+class SessionManager(BaseSessionManager):
     use_in_migrations = True
 
-    def encode(self, session_dict):
-        """
-        Returns the given session dictionary serialized and encoded as a string.
-        """
-        return SessionStore().encode(session_dict)
 
-    def save(self, session_key, session_dict, expire_date):
-        s = self.model(session_key, self.encode(session_dict), expire_date)
-        if session_dict:
-            s.save()
-        else:
-            s.delete()  # Clear sessions with no data.
-        return s
```

Comment:
Compare with [this](https://github.com/django/django/blob/7bb741d787ba360a9f0d490db92e22e0d28204ed/django/contrib/sessions/base_session.py#L9C1-L9C1) from Django:

```python
class BaseSessionManager(models.Manager):
    def encode(self, session_dict):
        """
        Return the given session dictionary serialized and encoded as a string.
        """
        session_store_class = self.model.get_session_store_class()
        return session_store_class().encode(session_dict)

    def save(self, session_key, session_dict, expire_date):
        s = self.model(session_key, self.encode(session_dict), expire_date)
        if session_dict:
            s.save()
        else:
            s.delete()  # Clear sessions with no data.
        return s
```